src/serac/physics/boundary_conditions/boundary_condition.hpp

```diff
@@ -132,16 +132,29 @@ class BoundaryCondition {
 
   /**
    * @brief "Manually" set the DOF indices without specifying the field to which they apply
-   * @param[in] dofs The indices of the DOFs constrained by the boundary condition
+   * @param[in] true_dofs The true vector indices of the DOFs constrained by the boundary condition
+   *
+   * @note This will set both the true and local internal dof index arrays.
    */
-  void setTrueDofs(const mfem::Array<int> dofs);
+  void setTrueDofs(const mfem::Array<int> true_dofs);
+
+  /**
+   * @brief "Manually" set the DOF indices without specifying the field to which they apply
+   * @param[in] local_dofs The local (finite element/grid function) indices of the DOFs constrained by the boundary
+   * condition
+   *
+   * @note This will set both the true and local internal dof index arrays.
+   */
+  void setLocalDofs(const mfem::Array<int> local_dofs);
```

Comment:
Should we have descriptions here what the difference between a truedof, localdof, and a dof is?